runtime/centrifuge/src/lib.rs

```diff
@@ -134,7 +134,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("centrifuge"),
 	impl_name: create_runtime_str!("centrifuge"),
 	authoring_version: 1,
-	spec_version: 1024,
+	spec_version: 1025,
```

Comment:
I guess this has been done for testing the migration, but should be in this PR?

Reply:
Yes, we wanted to numb that all the time directly after an upgrade according to William

Reply:
Yes, I should have bumped the spec versions after releasing.